toolchain/gnu/Makefile

```diff
@@ -0,0 +1,232 @@
+TOPDIR = $(CURDIR)
```

Comment:
Please add `# vim: tabstop=8 shiftwidth=8 noexpandtab:`. This is a common scheme used throughout the mimiker build system (see mimiker/build/).